shared/src/main/kotlin/com/egm/stellio/shared/model/ErrorResponse.kt

```diff
@@ -1,119 +0,0 @@
-package com.egm.stellio.shared.model
-
-import java.net.URI
-
-sealed class ErrorResponse(
```

Comment:
it should be the ultimate hierarchy for representing errors (https://old.arrow-kt.io/docs/patterns/error_handling/)